docs/tctl/namespace/register.md

```diff
@@ -57,6 +60,8 @@ tctl namespace register --description <value>
 ### `--global-namespace`
 
 Specifies whether a [Namespace](/concepts/what-is-a-namespace) is a [Global Namespace](/namespaces/#global-namespace).
+When enabled, it controls the creation of replication tasks on updates allowing the state to be replicated across Clusters.
+This is a read-only setting that can't be changed.
```

Comment:
```suggestion
This is a read-only setting and cannot be changed.
```
For consistency with earlier occurrence (currently line 36). Whether to use "cannot" or "can't" is under discussion.